app/base_scalers.py

```diff
@@ -63,6 +63,10 @@ def __init__(self, app_name, min_instances, max_instances):
         self.last_db_error = datetime.min
 
     def _init_db_uri(self):
+        if "SQLALCHEMY_DATABASE_URI" in os.environ:
+            self.db_uri = os.environ["SQLALCHEMY_DATABASE_URI"]
```

Comment:
Optional: You could do this as a one liner

`self.db_uri = os.environ["SQLALCHEMY_DATABASE_URI"].replace("postgresql://", "postgres://")`

which mirrors the equivalent in the API which does it as a one liner too

Reply:
Updated